recipes/nifti_clib/all/conanfile.py

```diff
@@ -0,0 +1,160 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class NiftiClibConan(ConanFile):
+    name = "nifti_clib"
+    description = "C libraries for NIFTI support"
+    license = "LicenseRef-LICENSE"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/NIFTI-Imaging/nifti_clib"
+    topics = ("image")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_nifti2": [True, False],
+        "use_cifti": [True, False],
+        "use_fslio": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "use_nifti2": True,
+        "use_cifti": False,  # seems to be beta?
+        "use_fslio": False  # Note in CMakeLists.txt: "If OFF, The copyright of this code is questionable for inclusion with nifti."
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("zlib/1.3.1")
+        if self.options.use_cifti:
+            self.requires("expat/2.6.2")
```

Comment:
```suggestion
        self.requires("zlib/[>=1.2.11 <2]")
        if self.options.use_cifti:
            self.requires("expat/[>=2.6.2 <3]")
```

These two libraries are some of the ones listed in https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#adherence-to-build-service so that version ranges are usable with them :)